nilearn/plotting/tests/test_displays.py

```diff
@@ -53,6 +56,58 @@ def test_tiled_slicer():
     slicer.close()
 
 
+def test_mosaic_slicer():
+    img = load_mni152_template()
+    # default cut_coords=None
+    slicer = MosaicSlicer.init_with_figure(img=img)
+    slicer.add_overlay(img, cmap=plt.cm.gray, colorbar=True)
+    # Forcing a layout here, to test the locator code
+    with tempfile.TemporaryFile() as fp:
+        slicer.savefig(fp)
+
+    # cut_coords as an integer
+    slicer = MosaicSlicer.init_with_figure(img=img, cut_coords=4)
+    slicer.add_overlay(img, cmap=plt.cm.gray, colorbar=True)
```

Comment:
I think it could be nice to add some additional checks in here. For example, we could check that everything is fine with `cut_coords`:

```python
for d in ['x', 'y', 'z']:
    assert d in slicer.cut_coords
    assert len(slicer.cut_coords[d]) == 4
```
WDYT?